src/app/pages/ArticlePage/index.jsx

```diff
@@ -1,6 +1,8 @@
 import ArticlePage from './ArticlePage';
 import applyBasicPageHandlers from '../utils/applyBasicPageHandlers';
 
+import withOptimizelyProvider from '#app/containers/PageHandlers/withOptimizelyProvider';
+
 export default applyBasicPageHandlers({
   addVariantHandling: true,
-})(ArticlePage);
+})(withOptimizelyProvider(ArticlePage));
```

Comment:
this function composition is a little weird. what about

```js
const OptimizelyArticle = withOptimizelyProvider(ArticlePage)
export default applyBasicPageHandlers({
  addVariantHandling: true,
})(OptimizelyArticle);
```

Reply:
Yep makes sense, refactored.